boards/nxp/frdm_rw612/doc/index.rst

```diff
@@ -190,6 +190,20 @@ with the application image directly, forming one single monolithic image.
 
 .. include:: ../../common/board-footer.rst
    :start-after: nxp-board-footer
+=======
+RTC Sub-Second Counter
+======================
+
+To use the RTC sub-second counter which is clocked at a 32kHZ rate, you first need to make the
+following modifications to the board hardware:
+
+1. Move the short on SJ21 from 1 and 2 to short 2 and 3.
+2. Move the short on SJ22 from 1 and 2 to short 2 and 3.
+
+After you make this change, the ENET will stop functioning on the board.
```

Comment:
i think this should go above the footer. cc @DerekSnell

Reply:
You are right. I have made the change.